src/Markdown.MAML/Parser/MarkdownParser.cs

```diff
@@ -148,7 +148,15 @@ private void InitializePatternList()
 
                         {   "italic",
                             @"\*(.+?)\*",
-                            this.CreateItalicSpan }
+                            this.CreateItalicSpan },
+
+                        {   "bold2",
+                            @"(?<![a-zA-Z0-9])__(.+?)__(?![a-zA-Z0-9])",
```

Comment:
I see. Then it's unlikely that existing bold and italic functions will suite, because they use group capturing and groups are different between these two regexes